tests/foreman/cli/test_host.py

```diff
@@ -302,6 +302,7 @@ def test_positive_search_all_field_sets():
 
 
 # -------------------------- CREATE SCENARIOS -------------------------
+@pytest.mark.e2e
```

Comment:
There are many tests in this test module which are good candidates for e2e. I dont have exact test cases to let you know but I think the one chosen here is covering less areas than what others are doing.

Reply:
added a few more, thanks